package-lock.json

```diff
@@ -36161,12 +35863,6 @@
         "is-typedarray": "^1.0.0"
       }
     },
-    "typescript": {
```

Comment:
I think this Lockfile now is fine. It gots cleaned up. Here for some reason is typescript for example